### ISSUE TITLE
Exception when building docs

### ISSUE THREAD

**Issue by ma-sadeghi** [sphinx-doc/sphinx#9889]
### Describe the bug

An exception is thrown when trying to build the docs (see Screenshot section for detailed log)	

### How to Reproduce

```
$ git clone https://github.com/PMEAL/OpenPNM.git
$ cd OpenPNM
$ git fetch origin fix_docs
$ git switch fix_docs
$ pip install -r requirements/pip_requirements.txt
$ pip install -r requirements/doc_requirements.txt
$ cd docs
$ make html
$ # an Exception will be thrown at this stage
```

### Your project

OpenPNM (see How to Reproduce for details on how to reproduce the issue)

### Screenshots

```python
# Sphinx version: 4.3.0
# Python version: 3.8.12 (CPython)
# Docutils version: 0.16 release
# Jinja2 version: 3.0.3
# Last messages:
#   reading sources... [ 27%] modules/generated/openpnm.core.Subdomain.update
#   reading sources... [ 27%] modules/generated/openpnm.core.Subdomain.values
#   reading sources... [ 27%] modules/generated/openpnm.utils
#   
#   ====================== slowest reading durations =======================
#   0.219 modules/generated/openpnm.core.Subdomain
#   0.162 modules/generated/openpnm.core.Base
#   0.133 examples/examples
#   0.111 modules/generated/openpnm.core.Subdomain.to_mask
#   0.108 modules/generated/openpnm.core.ModelsDict
# Loaded extensions:
#   sphinx.ext.mathjax (4.3.0) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/mathjax.py
#   sphinxcontrib.applehelp (1.0.2) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinxcontrib/applehelp/__init__.py
#   sphinxcontrib.devhelp (1.0.2) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinxcontrib/devhelp/__init__.py
#   sphinxcontrib.htmlhelp (2.0.0) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinxcontrib/htmlhelp/__init__.py
#   sphinxcontrib.serializinghtml (1.1.5) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinxcontrib/serializinghtml/__init__.py
#   sphinxcontrib.qthelp (1.0.3) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinxcontrib/qthelp/__init__.py
#   alabaster (0.7.12) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/alabaster/__init__.py
#   sphinx.ext.autodoc.preserve_defaults (1.0) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/autodoc/preserve_defaults.py
#   sphinx.ext.autodoc.type_comment (4.3.0) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/autodoc/type_comment.py
#   sphinx.ext.autodoc (4.3.0) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/autodoc/__init__.py
#   sphinx.ext.autosummary (4.3.0) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/autosummary/__init__.py
#   sphinx.ext.doctest (4.3.0) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/doctest.py
#   sphinx.ext.napoleon (4.3.0) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/napoleon/__init__.py
#   sphinx.ext.viewcode (4.3.0) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/viewcode.py
#   sphinx_copybutton (0.4.0) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx_copybutton/__init__.py
#   sphinx_panels (0.6.0) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx_panels/__init__.py
#   nbsphinx (0.8.7) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/nbsphinx.py
#   nbsphinx_link (1.3.0) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/nbsphinx_link/__init__.py
#   numpydoc (1.1.0) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/numpydoc/__init__.py
#   matplotlib.sphinxext.plot_directive (3.4.3) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/matplotlib/sphinxext/plot_directive.py
#   pydata_sphinx_theme (unknown version) from /home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/pydata_sphinx_theme/__init__.py
Traceback (most recent call last):
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/cmd/build.py", line 280, in build_main
    app.build(args.force_all, filenames)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/application.py", line 344, in build
    self.builder.build_update()
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/builders/__init__.py", line 294, in build_update
    self.build(to_build,
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/builders/__init__.py", line 308, in build
    updated_docnames = set(self.read())
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/builders/__init__.py", line 415, in read
    self._read_serial(docnames)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/builders/__init__.py", line 436, in _read_serial
    self.read_doc(docname)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/builders/__init__.py", line 476, in read_doc
    doctree = read_doc(self.app, self.env, self.env.doc2path(docname))
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/io.py", line 189, in read_doc
    pub.publish()
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/core.py", line 217, in publish
    self.document = self.reader.read(self.source, self.parser,
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/io.py", line 109, in read
    self.parse()
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/readers/__init__.py", line 77, in parse
    self.parser.parse(self.input, document)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/parsers.py", line 101, in parse
    self.statemachine.run(inputlines, document, inliner=self.inliner)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 170, in run
    results = StateMachineWS.run(self, input_lines, input_offset,
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/statemachine.py", line 241, in run
    context, next_state, result = self.check_line(
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/statemachine.py", line 459, in check_line
    return method(match, context, next_state)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 2769, in underline
    self.section(title, source, style, lineno - 1, messages)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 327, in section
    self.new_subsection(title, lineno, messages)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 393, in new_subsection
    newabsoffset = self.nested_parse(
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 281, in nested_parse
    state_machine.run(block, input_offset, memo=self.memo,
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 196, in run
    results = StateMachineWS.run(self, input_lines, input_offset)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/statemachine.py", line 241, in run
    context, next_state, result = self.check_line(
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/statemachine.py", line 459, in check_line
    return method(match, context, next_state)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 2342, in explicit_markup
    nodelist, blank_finish = self.explicit_construct(match)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 2354, in explicit_construct
    return method(self, expmatch)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 2096, in directive
    return self.run_directive(
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 2146, in run_directive
    result = directive_instance.run()
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/autodoc/directive.py", line 173, in run
    result = parse_generated_content(self.state, params.result, documenter)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/autodoc/directive.py", line 116, in parse_generated_content
    nested_parse_with_titles(state, content, node)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/util/nodes.py", line 335, in nested_parse_with_titles
    return state.nested_parse(content, 0, node, match_titles=1)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 281, in nested_parse
    state_machine.run(block, input_offset, memo=self.memo,
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 196, in run
    results = StateMachineWS.run(self, input_lines, input_offset)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/statemachine.py", line 241, in run
    context, next_state, result = self.check_line(
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/statemachine.py", line 459, in check_line
    return method(match, context, next_state)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 3005, in text
    self.section(title.lstrip(), source, style, lineno + 1, messages)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 327, in section
    self.new_subsection(title, lineno, messages)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 393, in new_subsection
    newabsoffset = self.nested_parse(
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 281, in nested_parse
    state_machine.run(block, input_offset, memo=self.memo,
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 196, in run
    results = StateMachineWS.run(self, input_lines, input_offset)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/statemachine.py", line 241, in run
    context, next_state, result = self.check_line(
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/statemachine.py", line 459, in check_line
    return method(match, context, next_state)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 2344, in explicit_markup
    self.explicit_list(blank_finish)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 2369, in explicit_list
    newline_offset, blank_finish = self.nested_list_parse(
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 318, in nested_list_parse
    state_machine.run(block, input_offset, memo=self.memo,
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 196, in run
    results = StateMachineWS.run(self, input_lines, input_offset)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/statemachine.py", line 241, in run
    context, next_state, result = self.check_line(
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/statemachine.py", line 459, in check_line
    return method(match, context, next_state)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 2647, in explicit_markup
    nodelist, blank_finish = self.explicit_construct(match)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 2354, in explicit_construct
    return method(self, expmatch)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 2096, in directive
    return self.run_directive(
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/docutils/parsers/rst/states.py", line 2146, in run_directive
    result = directive_instance.run()
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/autosummary/__init__.py", line 264, in run
    items = self.get_items(names)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/autosummary/__init__.py", line 389, in get_items
    sig = documenter.format_signature(show_annotation=False)
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/autodoc/__init__.py", line 1585, in format_signature
    sig = super().format_signature()
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/autodoc/__init__.py", line 1253, in format_signature
    result = self._find_signature()
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/autodoc/__init__.py", line 1201, in _find_signature
    docstrings = self.get_doc()
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/autodoc/__init__.py", line 1741, in get_doc
    return [prepare_docstring(docstring, ignore, tab_width) for docstring in docstrings]
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/ext/autodoc/__init__.py", line 1741, in <listcomp>
    return [prepare_docstring(docstring, ignore, tab_width) for docstring in docstrings]
  File "/home/amin/mambaforge/envs/sci/lib/python3.8/site-packages/sphinx/util/docstrings.py", line 75, in prepare_docstring
    lines = s.expandtabs(tabsize).splitlines()
AttributeError: 'property' object has no attribute 'expandtabs'
```

### OS

Linux (Ubuntu 20.04)

### Python version

3.8.12

### Sphinx version

4.3.0

### Sphinx extensions

'sphinx.ext.autodoc', 'sphinx.ext.autosummary', 'sphinx.ext.doctest', 'sphinx.ext.duration', 'sphinx.ext.mathjax', 'sphinx.ext.napoleon', 'sphinx.ext.viewcode', 'sphinx_copybutton', 'sphinx_panels', 'nbsphinx', 'nbsphinx_link', 'numpydoc', 'matplotlib.sphinxext.plot_directive'

**Comment by tk0miya** [comment]
Sorry for the inconvenience. This is already reported as #9879. It will be resolved in the next release v4.3.1 soon.
Thanks,

**Comment by ma-sadeghi** [comment]
@tk0miya Thank you so much, looking forward to it :)
